src/main/java/edu/kis/powp/jobs2d/command/visitor/VisitorFlipY.java

```diff
@@ -0,0 +1,31 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.util.Iterator;
+
+public class VisitorFlipY implements VisitorCommand {
+
+	@Override
+	public void visit(OperateToCommand operateToCommand) {
+		operateToCommand.setPosX(operateToCommand.getPosX() * (-1));
+	}
+
+	@Override
+	public void visit(SetPositionCommand setPositionCommand) {
+		setPositionCommand.setPosX(setPositionCommand.getPosX() * (-1));
```

Comment:
getPosX() * (-1) seems to be a minor code duplication (also in visitOperateTo) but it is present in all transformation visitors. There exist more general, flexible, and reusable solution!

Reply:
refactored - added an additional abstraction layer